### DISCUSSION TITLE
are there any prerequisites in windows I'm not aware of?

### DISCUSSION THREAD

**Discussion by Vijartal** [FlareSolverr/FlareSolverr#555] [Q&A]
just tried to use my flaresolverr/fanficfare usb on a different pc "new laptop"
are there any prerequisites in windows I'm not aware of? like .net/etc
this is what I'm getting. V2.2.9 "there's a number of similar errors beforehand"
2022-10-21T20:26:48+11:00 ERROR REQ-4 Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

**Comment by ilike2burnthing** [comment]
No.

https://github.com/FlareSolverr/FlareSolverr/issues/536

**Comment by Chezziex** [comment]
I'm having the same errors .  Is there something I'm missing to get this working?

**Reply by ilike2burnthing** [reply]
See the linked issue above to follow its progress. You can try the beta mentioned there.

**Comment by ngosang** [comment]
FlareSolverr 3.0.0 is out and it has other requirements. Check out the readme.
